src/main/java/mate/jdbc/util/Injector.java

```diff
@@ -1,4 +1,4 @@
-package mate.jdbc.lib;
+package mate.jdbc.util;
```

Comment:
why did you change the package for Injector? This is why you can't use `lib` folder without imports